### PR TITLE
Update code readability

### PR DESCRIPTION
**Pull request by elsh**
Use explicit labels, use .contains for better readability, etc.

### CODE REVIEW

**Comment by elsh** [comment]
@swift-ci smoke test

File: Sources/PackageGraph/PackageGraph+Loading.swift [hunk 1]
```diff
@@ -662,11 +664,11 @@ private func resolveModuleAliases(with packageBuilders: [ResolvedPackageBuilder]
     }
 
     // Track targets that need module aliases for each package
-    for packageBuilder in packageBuilders {
-        for prod in packageBuilder.package.products {
-            var list = prod.targets.map{$0.dependencies}.flatMap{$0}.compactMap{$0.target?.name}
-            list.append(contentsOf: prod.targets.map{$0.name})
-            aliasTracker.addAliasesForTargets(list, for: prod.name, in: packageBuilder.package.identity)
+    packageBuilders.forEach { packageBuilder in
+        packageBuilder.package.products.forEach { produdct in
```
Comment (tomerd) [inline Sources/PackageGraph/PackageGraph+Loading.swift:668]: @abertelrud told me in the past that they preferred for ... in in SwiftPM code base

File: Sources/PackageGraph/PackageGraph+Loading.swift [hunk 2]
```diff
@@ -687,11 +689,11 @@ private class ModuleAliasTracker {
...
 
     func addAlias(_ alias: String,
-                  of targetName: String,
-                  for product: String,
-                  from originPackage: PackageIdentity,
-                  in parentPackage: PackageIdentity) {
-        let model = ModuleAliasModel(name: targetName, alias: alias, originPackage: originPackage, parentPackage: parentPackage)
+                  target: String,
+                  product: String,
+                  originPackage: PackageIdentity,
```
Comment (tomerd) [inline Sources/PackageGraph/PackageGraph+Loading.swift:694]: is originPackage the package from which the target / product is from?

**Review by tomerd — COMMENTED** [review]
these are great improvements to readability - thank you. one question to @abertelrud about iteration style

File: Sources/PackageGraph/PackageGraph+Loading.swift (hunk 2 above)
Comment (elsh) [reply Sources/PackageGraph/PackageGraph+Loading.swift:694]: yes

File: Sources/PackageGraph/PackageGraph+Loading.swift (hunk 1 above)
Comment (elsh) [reply Sources/PackageGraph/PackageGraph+Loading.swift:668]: ok will revert back to `for...`

**Comment by elsh** [comment]
@swift-ci smoke test

**Review by tomerd — APPROVED** [review]

**Comment by tomerd** [comment]
@swift-ci please smoke test macOS

**Pull request merged**